packages/firestore/test/unit/local/indexeddb_schema.test.ts

```diff
@@ -210,3 +237,39 @@ describe('IndexedDbSchema: createOrUpgradeDb', () => {
     );
   });
 });
+
+describe('IndexedDb: allowTabSynchronization', () => {
+  if (!IndexedDbPersistence.isAvailable()) {
+    console.warn('No IndexedDB. Skipping allowTabSynchronization tests.');
+    return;
+  }
+
+  beforeEach(() => SimpleDb.delete(INDEXEDDB_TEST_DATABASE));
+
+  after(() => SimpleDb.delete(INDEXEDDB_TEST_DATABASE));
+
+  it('rejects access when synchronization is disabled', () => {
+    return withPersistence('clientA', async db1 => {
+      await expect(db1.start(/*synchronizeTabs=*/ false)).to.eventually.be
+        .fulfilled;
```

Comment:
Out of curiosity, is `expect(...).to.eventually.be.fulfilled` adding anything here?  Couldn't you just `await db1.start(...)` ?  If it fails, it'll throw an exception, which will fail the test.

I don't care strongly if this is intentional though...

Reply:
Hm, I just used it because I wanted to use  the `rejectedWith` counterpart. It doesn't add any value though, beside the increase in complexity. Removed.